doc/howto/generic-install-from-source.rst

```diff
@@ -194,3 +201,20 @@ the :file:`html` directory to a suitable location and make sure that location is
 as ``/doc`` on the web server.  If using the example Apache configuration
 (:file:`apache.conf.example`), there is a define named ``documentation_path``,
 which can be set to point to this file system location.
+
+Building CSS assets
+-------------------
+
+If you want to modify the styling, you can do so by editing the SCSS files
+in the :file:`nav/web/static/scss` directory. To build the CSS assets, you will
+need to have Node.js and npm installed. Once you have these installed, you can
+run the following command to build the CSS assets::
+
+  npm install
+  npm run build:sass
+
+This will build the CSS assets and place them in the :file:`nav/web/static/css`
+directory. If you want to watch for changes in the SCSS files and rebuild the
+CSS assets automatically, you can run the following command instead::
+
+  npm run watch:sass
```

Comment:
I think this bit is out of scope for regular installation: Normal users should not be interested in this for production use, it belongs in the hacking guide.